### PR TITLE
Splits the project into `core` and `cli` modules

### PR DESCRIPTION
**Pull request by andreban**
- Uses `lerna` to manage modules
- Adds README.md for each module
- Updates version to 0.4.0

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -28,58 +28,11 @@ feature requests, and contribute with pull requests, if possible.
...
-When building the project for the first time, the Android Build Tools will need to be installed.
-The tool will inkove the installation process for the build tools. Make sure to read and accept
-the license agreement before proceeding.
-
-As a result of the build step, the tool will generate a signed APK (`app-release-signed.apk`)
-that can be uploaded to the Play Store. You will also need to deploy a Digital Asset Links file to
-validate your domain. The
-[TWA Quick Start Guide](https://developers.google.com/web/updates/2019/08/twas-quickstart#creating-your-asset-link-file)
-explains how to extract the information needed to generate it.
+- **[llama-pack-core](./packages/core):** a javascript library that for generating, building and
```
Comment (PEConn) [inline README.md]: nit: remove "that"

File: README.md [hunk 2]
```diff
@@ -28,58 +28,11 @@ feature requests, and contribute with pull requests, if possible.
...
+updating TWA projects.
+- **[llama-pack-cli](./packages/cli):** a command-line version for llama-pack.
```
Comment (PEConn) [inline README.md]: A command-line "version" sounds a bit odd.

"llama-pack core wrapped for the command line" maybe?

File: package.json [hunk 3]
```diff
@@ -24,33 +22,17 @@
...
+    "@llama-pack/core": "file:packages/core",
+    "@llama-pack/cli": "file:packages/cli"
   },
   "devDependencies": {
     "@typescript-eslint/eslint-plugin": "^2.9.0",
     "@typescript-eslint/parser": "^2.9.0",
     "eslint": "^6.6.0",
     "eslint-config-google": "^0.14.0",
-    "jasmine": "^3.5.0"
+    "jasmine": "^3.5.0",
```
Comment (PEConn) [inline package.json]: So the tests are in the parent package?

File: packages/core/src/index.ts [hunk 4]
```diff
@@ -0,0 +1,34 @@
...
+ *  distributed under the License is distributed on an "AS IS" BASIS,
+ *  WITHOUT WARRANTIES OR CONDITIONS OF ANY KIND, either express or implied.
+ *  See the License for the specific language governing permissions and
+ *  limitations under the License.
+ */
+
+import {AndroidSdkTools} from './lib/androidSdk/AndroidSdkTools';
+import {Config} from './lib/Config';
+import {GradleWrapper} from './lib/GradleWrapper';
+import Log from './lib/Log';
```
Comment (PEConn) [inline packages/core/src/index.ts]: Why no {}?

File: README.md (hunk 2 above)
Comment (andreban) [reply README.md]: "command-line interface for llama-pack"?

File: packages/core/src/index.ts (hunk 4 above)
Comment (andreban) [reply packages/core/src/index.ts]: `Log` is using `export default Log`, while others simple use `export`. Maybe I should change `Log` to just use `export`, but likely in another PR.

File: package.json (hunk 3 above)
Comment (andreban) [reply package.json]: We don't cleaned-up the dependencies in root. We can re-add later if we end up needing.

**Pull request merged**